subprojects/instant-execution/src/main/kotlin/org/gradle/instantexecution/InstantExecutionException.kt

```diff
@@ -81,8 +83,8 @@ class TooManyInstantExecutionProblemsException internal constructor(
     problems: List<PropertyProblem>,
     htmlReportFile: File
 ) : InstantExecutionProblemsException(
-    "Maximum number of instant execution problems has been reached.\n" +
-        "This behavior can be adjusted via -D${SystemProperties.maxProblems}=<integer>.",
+    "Maximum number of configuration cache problems has been reached.\n" +
+        "This behavior can be adjusted via -D${ConfigurationCacheMaxProblemsOption.PROPERTY_NAME}=<integer>.",
```

Comment:
Same here, perhaps we link to the user manual instead of suggesting only one of several options in the console output.